application/config/config-defaults.php

```diff
@@ -804,6 +804,8 @@
 
 $config['pluginWhitelist'] = [];
```

Comment:
Can you add comment on this one ? For https://github.com/LimeSurvey/LimeSurvey/blob/64fb578c5f0fddc0ea89fe4ede27cd44a3f29d82/application/config/config-defaults.php#L805 something like 

> User plugin list allowed to be installed. This don't disallow core plugin, but disable already installed plugin action.